content/glossary/runoff.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Runoff
+---
+
+Water that is discharged into the streams and largely results from precipitation and melting of snow.
+
+Runoff projections on Cal-Adapt are produced using a land surface and hydrology model known as the [Variable Infiltration Capacity](/help/glossary/) (VIC) model.
```

Comment:
add glossary link to vic

Reply:
fixed via https://github.com/berkeley-gif/caladapt-website-2021/pull/125/commits/69cb77de4d70c5bd1c8af193d3592b86a0a44ea7